client/src/pages/CreateUserProfile.js

```diff
@@ -235,6 +235,14 @@ const CreateProfile = ({ email, history }) => {
     createUserFormDispatch({ type: CREATE_USER });
     try {
       const { email, ...body } = formData;
+      /* TEMP location until geolocation service id done */
+      body.location = {
+        address: 'New York, NY, USA',
```

Comment:
ah ok cool didn't know it was like this, so does that mean, `Address` field is what can be displayed? Or is that street address and it still has to be mergd together with whatever other info is in here @joshmorel

Reply:
Based on what some of the designs and a question in Slack my plan was to equate address to Google formatted_address which could be of varying degrees of granularity but is never just street address (e.g. "44 Bank St").

So for example from this: 
https://app.zeplin.io/project/5ecaed1b2522d9280f1a4792/screen/5ed56900260b9c1e9ddf8b2f

![image](https://user-images.githubusercontent.com/10546720/83954227-5d38bb00-a815-11ea-8a9f-22a899a90c1f.png)


The Google address is:
"Neukölln, Berlin, Germany"

And the structured location is:
"country": "DE"
"state": "Berlin",
"city": "Berlin",
"neighborhood": "Neukölln"
"coordinates": [13.4445071, 52.4407709]

Based on autocomplete suggestions you could go more granular. This might be confusing so there's way to limit that I think we could resolve post soft launch.